connectors/class-connector-blogs.php

```diff
@@ -144,14 +147,38 @@ public function callback_wpmu_new_blog( $blog_id ) {
 				'stream'
 			),
 			array(
-				'site_name' => $blog->blogname,
+				'site_name' => ! empty( $blogname ) ? $blogname : 'Site %d',
 			),
 			$blog_id,
-			sanitize_key( $blog->blogname ),
+			sanitize_key( $blogname ),
 			'created'
 		);
 	}
 
+	/**
+	 * A site has been deleted from the database.
+	 *
+	 * @action wp_deleted_site
+	 *
+	 * @param WP_Site $old_site  Deleted site object.
+	 */
+	public function callback_wp_delete_site( $old_site ) {
+		$this->log(
+			/* translators: %s: site name (e.g. "FooBar Blog") */
+			_x(
+				'"%s" site was deleted',
+				'1. Site name',
+				'stream'
+			),
+			array(
+				'site_name' => $old_site->blogname,
+			),
+			$old_site->blog_id,
+			sanitize_key( $old_site->blogname ),
+			'deleted'
+		);
+	}
+
```

Comment:
@dero @johnbillion This callback was added to resolve #1175. Please take a look.

Reply:
Thanks @kidunot89 -- this callback will add an appropriate log entry, but AFAICT won't prevent the related "User removed" etc. entries from being logged, which can really clutter the log when a blog with dozens of users is removed.

I'm OK approving this PR as is, but I think we should keep #1175 open until we're able to bundle up those excessive individual log entries somehow. I'm thinking perhaps we find a way to just log "XYZ users were removed from the ABC site."?

What do you guys think?